engine/build.gradle

```diff
@@ -216,26 +214,31 @@ group = 'org.terasology.engine'
 println "Version for $project.name loaded as $version for group $group"
 
 // This version info file actually goes inside the built jar and can be used at runtime
-task createVersionInfoFile {
-    inputs.property('dateTime', startDateTimeString)
-    onlyIf { env.BUILD_URL != null }
```

Comment:
@Cervator This task had an "only if $BUILD_URL is set" condition on it. Is that important to keep? i.e. is a properties file with a bunch of missing entries worse than no properties file at all?

Reply:
So that's the guard that means it normally only runs in Jenkins, which sets the env vars that get read. As for whether an incomplete file is worse than empty - unsure. It might list gibberish in the main menu? The file also gets attached in the open in Jenkins so the launcher can read it, but I'm not sure about the details there